.github/workflows/release-build.yml

```diff
@@ -220,7 +228,7 @@ jobs:
         echo "Building shared library with ${BUILD_GEN} for ${BUILD_ARCH}"
         choco install -y swig
         mkdir build && cd build
-        cmake -G "${BUILD_GEN}" -A "${BUILD_ARCH/x86/Win32}" -DCMAKE_BUILD_TYPE=Release -DENABLE_PACKAGE_BUILD=ON -DDISABLE_STATIC_LIB_INSTALL=ON -DSTATIC_STANDARD_LIB=ON -DBUILD_APP_EXECUTABLES=OFF -DBUILD_HELICS_EXAMPLES=OFF -DBUILD_TESTING=OFF .. || true
+        cmake -G "${BUILD_GEN}" -A "${BUILD_ARCH/x86/Win32}" -DCMAKE_BUILD_TYPE=Release -DHELICS_ENABLE_PACKAGE_BUILD=ON -DHELICS_DISABLE_STATIC_LIB_INSTALL=ON -DSTATIC_STANDARD_LIB=ON -DHELICS_BUILD_APP_EXECUTABLES=OFF -DHELICS_BUILD_APP_LIBRARY=OFF -DHELICS_BUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF .. || true
```

Comment:
The disable static Lib install option has been removed completely I think, so we should be able to remove it

Reply:
Do you want the two occurrences of `HELICS_DISABLE_STATIC_LIB_INSTALL` removed?

The only place where it is used and has an effect is https://github.com/GMLC-TDC/HELICS/blob/develop/config/cmake/addlibzmq.cmake

There is also the place where it is set to ON in https://github.com/GMLC-TDC/HELICS/blob/develop/CMakeLists.txt#L121

Reply:
lets remove them here in this PR,  I will remove the other extraneous references in the warning_fixes PR since I am already touching some of those files.

Reply:
Okay, the builds no longer use that option.